### PR TITLE
docs: restructure to trg1 and add policies and attributes

### PR DESCRIPTION
**Pull request by Phil91**
## Description

* adjust documentation structure to align with TRG1
* add documentation to add policies and policy attributes

## Why

To align with trg 1 and give the overview of how to add new policies and policy attributes.

## Issue

Refs: #178 #197

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added copyright and license headers, footers (for .md files) or files (for images)

### CODE REVIEW

**Review by evegufy — CHANGES_REQUESTED** [review]
please add /docs/README.md and align with https://github.com/eclipse-tractusx/ssi-authority-schema-registry/pull/71 where it makes sense

**Comment by Phil91** [comment]
> [eclipse-tractusx/ssi-authority-schema-registry#71](https://github.com/eclipse-tractusx/ssi-authority-schema-registry/pull/71)

@evegufy did add the readme

File: docs/architecture/Index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+This documentation follows the arc42 template to provide a comprehensive overview of the SSI Authority & Schema Registry App.
+
+## Table of Content
+
+- [01. Introduction and Goals (To be added.)](./Index.md)
+- [02. Architecture Constraints](./02_architecture_constraints.md)
+- [03. Context and Scope](./03_system_scope_and_context.md)
+- [04. Solution Strategy](./04_solution_strategy.md)
+- [05. Building Block View](./05_building_block_view.md)
+- [06. Runtime View (To be added.)](./Index.md)
```
[the remark this answers is not in the document]
Comment (Phil91) [reply docs/architecture/Index.md]: adjusted

File: docs/architecture/Index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Arc42
+
+This documentation follows the arc42 template to provide a comprehensive overview of the SSI Authority & Schema Registry App.
```
[the remark this answers is not in the document]
Comment (Phil91) [reply docs/architecture/Index.md]: adjusted

File: docs/architecture/Index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Phil91) [reply docs/architecture/Index.md]: created a pr for the fix in asr: https://github.com/eclipse-tractusx/ssi-authority-schema-registry/pull/87

File: docs/known-knowns/known-issues-and-limitations.md
Comment (evegufy) [inline docs/known-knowns/known-issues-and-limitations.md:1]: can be removed due to https://github.com/eclipse-tractusx/policy-hub/issues/186#issuecomment-2293670726
for future reference, I'd put it under docs/admin

File: docs/admin/add-data-process/index.md
Comment (evegufy) [inline docs/admin/add-data-process/index.md:1]: might make sense to rename docs/admin/add-policies/add-policies.md

**Review by evegufy — CHANGES_REQUESTED** [review]

File: docs/known-knowns/known-issues-and-limitations.md
Comment (Phil91) [reply docs/known-knowns/known-issues-and-limitations.md:1]: adjusted

**Review by evegufy — APPROVED** [review]

**Pull request merged**